jerry-core/ecma/base/ecma-globals.h

```diff
@@ -204,7 +204,7 @@ typedef enum
 
   ECMA_INTERNAL_PROPERTY_NATIVE_HANDLE, /**< native handle associated with an object */
   ECMA_INTERNAL_PROPERTY_FREE_CALLBACK, /**< object's native free callback */
-
+  ECMA_INTERNAL_PROPERTY_NOT_FOUND_AND_STOP, /**< the property is not found and no more search */
```

Comment:
We discussed that a["-1.1"] is undefined for typed arrays, but it has this property with value undefined or has not? If it has not then why it aborts the search?